### PR TITLE
Add Turkish translations

### PR DESCRIPTION
**Pull request by enesTufekci**
Hello there, i would love to help/maintain Turkish translations.

**here are some open source contributions;**
https://github.com/enesTufekci/flex-carousel
https://github.com/enesTufekci/color-bucket

**react work;**
https://github.com/systemaccounting/mxfactorial

**Production**
https://moebel.de/wohnen
http://arhiva.muzeultaranuluiroman.ro:3003/arhiva (Under construction)

### CODE REVIEW

**Comment by tesseralis** [comment]
Thanks for volunteering! I'll keep this open so others can join in as well.

**Comment by helloanil** [comment]
Count me in too

**Comment by tesseralis** [comment]
Yay! @enesTufekci say the word and I can merge this in so you two can get started.

File: langs/tr.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    "name": "Turkish",
+    "code": "tr",
+    "maintainers": ["enesTufekci"]
```
Comment (helloanil) [inline langs/tr.json]:
```suggestion
    "maintainers": ["enesTufekci", "helloanil"]
```

Comment (helloanil) [reply langs/tr.json]: Hey @enesTufekci, can you also add me to the maintainers list by commiting this suggested change? 💪

**Comment by ozcanzaferayan** [comment]
Hi. I can help Turkish translation for this project.

My wikipedia works are:
https://tr.wikipedia.org/wiki/Dart_(programlama_dili)
https://tr.wikipedia.org/wiki/SMPP
https://tr.wikipedia.org/wiki/A%C3%A7%C4%B1k_ve_Gizli_Anahtarl%C4%B1_%C5%9Eifreleme

Also currently writing Turkish articles on devnot.com:
http://devnot.com/author/zafer-ayan/

ScreenToGif app translation:
https://www.screentogif.com/?l=tr_tr

**Pull request merged**